docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -49,6 +49,17 @@ Breaking changes
   :py:meth:`~pvlib.pvsystem.PVSystem.calcparams_desoto` and
   :py:meth:`~pvlib.pvsystem.PVSystem.calcparams_cec` (:issue:`1118`, :pull:`1222`)
 
+* Switched the order of the outputs from the PSM3 iotools, notably
+  :py:func:`~pvlib.iotools.get_psm3` and :py:func:`~pvlib.iotools.read_psm3`
+  (:issues:`1245`, :pull`1268`)
```

Comment:
```suggestion
  (:issue:`1245`, :pull:`1268`)
```

and same below

Reply:
I guess this is why you shouldn't code after 10 pm... sorry for making you look at this sloppy code